editor/src/editor.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react";
+import AceEditor from "react-ace";
+import "ace-builds/src-noconflict/mode-css";
+import "ace-builds/src-noconflict/theme-monokai";
+
+const Editor = () => {
+  const [styles, setStyles] = useState("");
+
+  const onChange = (newStyle) => {
+    setStyles(newStyle);
+  };
+  return (
+    <>
+      <style>{styles}</style>
+      <AceEditor
```

Comment:
I didn't try using Ace Editor, but this looks really clean. Did you have experience with it before this project?